### DISCUSSION TITLE
Closing current tab programatically from a button

### DISCUSSION THREAD

**Discussion by CovantaDataManagement** [Budibase/budibase#5104] [Help] [answered]
I am trying to figure out how to close the current app tab using an HTML tag something like this:
<a href="javascript:closeWindow();">close</a>
I know there is an X but the user is rigid on this.  How might I do this?

**Comment by shogunpurple** [comment, marked as answer]
Hey @CovantaDataManagement - this will work for you:

- Create an embed component
- Add the following code

```
<button onclick="window.close()">
Close
</button>
```

Thanks!

**Reply by CovantaDataManagement** [reply]
Good Morning!
Was there a code snippet that was supposed to be in the response?

**Reply by shogunpurple** [reply]
Whoops - sorry, I copy and pasted from your comment I replied to here:
https://github.com/Budibase/budibase/discussions/4831#discussioncomment-2446227

Updated this ^

**Reply by CovantaDataManagement** [reply]
AWESOME!!!!!
